tasks/unix/add-jenkins-user.yml

```diff
@@ -1,7 +1,4 @@
 ---
-- name: Include platform-specific defaults
-  ansible.builtin.include_vars: "{{ role_path }}/vars/{{ ansible_os_family | lower }}.yml"
-
```

Comment:
Removing these lines doesn't seem related to adding the `swarm_client_create_user` default. But these lines also seem to be unused for quite some time... (forever?)

Reply:
The `include_vars` is necessary if `add-jenkins-user.yml` is called by the playbook via `include_role` (which I think we used to do). But with the introduction of the `swarm_client_create_user` variable, it should never be necessary to call this task file directly, which is why I felt that this change belonged to that commit.

Reply:
> The `include_vars` is necessary if `add-jenkins-user.yml` is called by the playbook via `include_role`

Why? I don't see what variables from these files are needed.

Reply:
You're right, it might be for historical reasons, then.